tasks/appraisal-tab.yml

```diff
@@ -9,9 +9,6 @@
 - name: "Fix install path of node vs nodejs via symlink"
   file: src=/usr/bin/nodejs dest=/usr/bin/node state=link
 
-- name: "Install bower"
-  npm: name=bower global=yes
-
-- name: "Install bower dependencies"
-  bower: path="{{ archivematica_src_dir }}/archivematica/src/dashboard/src/external/appraisal-tab/"
+- name: "Install appraisal tab dependencies"
+  npm: path="{{ archivematica_src_dir }}/archivematica/src/dashboard/src/external/appraisal-tab/"
```

Comment:
Should/can this run `npm install` instead?  That should install the dependencies and build the webpack.